src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -963,11 +963,12 @@ private InstructorFeedbackResultsQuestionTable buildQuestionTableAndResponseRows
             }
             
             // If question specific sorting is not needed, responses are sorted
-            // by default order (first by team name, then by display name)
+            // in the following order (first by recipient name, then by team name,
+            // then by display name)
             if (questionDetails.isQuestionSpecificSortingRequired()) {
                 Collections.sort(responseRows, questionDetails.getResponseRowsSortOrder());
             } else {
-                responseRows = InstructorFeedbackResultsResponseRow.sortListWithDefaultOrder(responseRows);
+                responseRows = InstructorFeedbackResultsResponseRow.sortListByRecipientNameThenDefaultOrder(responseRows);
```

Comment:
@kanghj Thanks for the clarification. Just a few more question, constant sum here refers to those questions which allocates points to each option and sums up to 100 right? Also, this issue refers to changing the display of the constant sum statistics by default or needs to be invoked in some way?